accounts/forms.py

```diff
@@ -0,0 +1,31 @@
+# coding: utf-8
+
+from django import forms
+
+from accounts.models import Account
+
+
+class AccountCreateForm(forms.ModelForm):
+
+    password2 = forms.CharField(widget=forms.PasswordInput())
```

Comment:
Why you override password2 here and do `self.fields['password'].widget = forms.PasswordInput()` below? Couldn't both be overriden in the same place?